.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+name: build
+on:
+  push:
+    branches: [main, release/**]
+
+jobs:
+  dist:
+    name: Create Distribution
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+      - name: "Prepare Artifacts"
+        run: |
+          pip install wheel
+          pip install build
+          python -m build --wheel
```

Comment:
running this with no arguments is preferable since you will get both sdist and wheel